recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -9,9 +9,9 @@ class JfalcouEveConan(ConanFile):
     description = ("Expressive Velocity Engine - reimplementation of the old "
                    "Boost.SIMD on C++20"
                    )
-    homepage = "https://github.com/jfalcou/eve"
+    homepage = "https://jfalcou.github.io/eve/"
     topics = ("c++", "simd")
-    license = "MIT"
+    license = "BSL-1.0"
```

Comment:
You will need to do something more complicated fore those pesky enterprise user that use the client to search
this will change it for old versions which is no accurate (though i doubt it would be a problem)

I would set both here and they change it based on the version like

https://github.com/conan-io/conan-center-index/blob/924830c2a020224466b594ec5d7d2bb15157dd46/recipes/eigen/all/conanfile.py#L25-L29

Reply:
Ok will have a run at this today

Reply:
OK, I looked at the Eigen example but it looks like it just use an option to select the licence.
How can I test for my own version within configure ?

Reply:
`self.version` should be available.